src/components/timeline/timeline-event/timeline-event.component.html

```diff
@@ -0,0 +1,15 @@
+<li class="timeline-inverted">
+    <div [ngClass]="['timeline-badge', badgeColor]">
+        <span>{{ badgeTitle }}</span>
+    </div>
+    <div class="timeline-panel">
+        <div class="timeline-heading">
+            <h4>{{ title }}</h4>
+            <span *ngIf="timestamp" class="timeline-heading-timestamp">
+                <i class="hpe-icon hpe-document-time"></i>
+                <span>{{ timestamp.toLocaleString() }}</span>            

```

Comment:
If it has been decided to use a date object here then consider using the Angular [DatePipe](https://angular.io/api/common/DatePipe) instead as it can memoize the result.

Reply:
Perhaps the whole content of the span should be templateable. I'm also not a fan of the unconfigurable hpe icon here.

Reply:
Good point, should we even force them to have a title? It might be better just to allow them to have complete control over the entire content and just have the ng-content here?

Reply:
Yes, especially the random use of a rank 4 header. Let's just template the whole thing and drop the `title` and `timestamp` properties. I was looking at GitHub's issue tracker which is a good timeline example; they use relative dates a lot, so we ought to give authors the freedom to use that, or any other date presentation that suits the application.